STUDIO/org.openl.rules.repository.git/src/org/openl/rules/repository/git/CommitMessageParser.java

```diff
@@ -59,47 +57,40 @@ private static String buildPattern(String template) {
                 }
                 if (matched) {
                     if (start < end) {
-                        //quote plain text
+                        // quote plain text
                         builder.append("\\Q").append(template, start, end).append("\\E");
                     }
                     start = pos;
                     final boolean firstTokenOccurrence = tokenCounter[tokenIdx] == 0;
                     tokenCounter[tokenIdx]++;
-                    //open group
+                    // open group
                     builder.append("(?");
                     if (!firstTokenOccurrence) {
-                        //make non-capturing group
+                        // make non-capturing group
                         builder.append(':');
                     }
                     switch (token) {
                         case COMMIT_TYPE_TOKEN:
                             if (firstTokenOccurrence) {
-                                //set group name
+                                // set group name
                                 builder.append("<commitType>");
                             }
                             builder.append(COMMIT_TYPES);
                             break;
                         case USER_MESSAGE_TOKEN:
                             if (firstTokenOccurrence) {
-                                //set group name
+                                // set group name
                                 builder.append("<message>");
                             }
                             builder.append(".*");
                             break;
-                        case USERNAME_TOKEN:
-                            if (firstTokenOccurrence) {
-                                //set group name
-                                builder.append("<author>");
-                            }
-                            builder.append(".+");
-                            break;

```

Comment:
It will break backward capability, I think you should leave an ability to parse old commits

Reply:
Fixed.